src/Orchard.DisplayManagement/Theming/ThemeManager.cs

```diff
@@ -47,7 +47,7 @@ public class ThemeManager : IThemeManager
                 foreach (var theme in themeResults)
                 {
                     var t = _extensionManager.GetExtension(theme.ThemeName);
-                    if (t.Exists)
+                    if (t?.Exists ?? false)
```

Comment:
@jtkech Can you revert this, Get Extension should return a NotFoundExtension class, with Exists = False. Get Extension should not return null.